### PR TITLE
[Project Submission] FileDrive

### PR DESCRIPTION
**Pull request by laurarenpanda**

### CODE REVIEW

File: projects/application/filedrive.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Previews
+### Homepage: 
+![PreviewOne](https://user-images.githubusercontent.com/74699939/108585788-4fb46880-7385-11eb-8c36-682da3930120.png)
+### DataBank
+![PreviewTwo](https://user-images.githubusercontent.com/74699939/108585789-55aa4980-7385-11eb-94cb-48fb116a217c.png)
+### Detailed information of certain dataset 
+![PreviewThree](https://user-images.githubusercontent.com/74699939/108585792-5a6efd80-7385-11eb-9e12-bca4488a8ece.png)
+### Detailed information of each file
+![PreviewFour](https://user-images.githubusercontent.com/74699939/108585793-5fcc4800-7385-11eb-8e57-47ccd2516e9f.png)
+
```
Comment (jennijuju) [inline projects/application/filedrive.md:18]: Could you please add 

```
## Project Status
<!--brainstorming, fundraising, under development, beta, shipped, etc-->
```

File: projects/application/filedrive.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Offline Deal 
+      - support data from IPFS network
+      - support multiple deal proposals in one PublishStorageDeal message
+      - add config for Base Fee
+- Data Retrival
+      - use IPLD data structure instead of tar
+      - retrieve small files in IPLD graph througth IPFS network
+
+## Contact Us
```
Comment (jennijuju) [inline projects/application/filedrive.md:40]: Please link the discussion thread

**Review by jennijuju — CHANGES_REQUESTED** [review]

**Comment by jennijuju** [comment]
Do you want your updates to be featured in Filecoin newsletter?

**Comment by laurarenpanda** [comment]
Yes, we'd like to. Please put our updates in Filecoin newsletter. Thanks.

Jennifer <notifications@github.com> 于2021年2月24日周三 上午9:06写道：

**Review by jennijuju — APPROVED** [review]

**Pull request merged**